src/System.Reflection.Metadata/src/System/Reflection/Metadata/PortablePdb/MethodBody.cs

```diff
@@ -37,5 +36,16 @@ public BlobHandle SequencePoints
                 return _reader.MethodBodyTable.GetSequencePoints(Handle);
             }
         }
+
+        /// <summary>
+        /// Returns local signature handle.
+        /// </summary>
+        public StandaloneSignatureHandle LocalSignature
+        {
+            get
+            {
+                return StandaloneSignatureHandle.FromRowId(_reader.GetBlobReader(SequencePoints).ReadCompressedInteger());
+            }
+        }
```

Comment:
OK, I think we can make it Blob (or Data) and add GetSequencePointReader() method so that it is obvious how to get sequence points out of it.

MethodDebugInfo sounds good. We should also rename CustomDebugInformation to CustomDebugInfo if we are going to use the short "Info" form.

Will leave renaming for another time as you suggest :)